### PR TITLE
PR: Codebase for DappLooker Go SDK

### PR DESCRIPTION
**Pull request by 0xSumitBanik**
## Overview

This pull request introduces the Go Software Development Kit (SDK) for DappLooker under the go directory. The SDK enables seamless access to DappLooker's chart data through the Web3 environment. Leveraging the power of Go, developers can effortlessly integrate DappLooker's analytics platform into their projects.

### CODE REVIEW

File: go/dapplooker/dapplookerCharts.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	return &DappLookerChartsAPI{
+		chartUUID: chartUUID,
+		apiKey:    apiKey,
+		format:    format,
+	}
+}
+
+// GetChartData retrieves chart data from the DappLooker API.
+func (api *DappLookerChartsAPI) GetChartData() (map[string]interface{}, error) {
+	url := fmt.Sprintf("https://api.dapplooker.com/chart/%s?api_key=%s&output_format=%s", api.chartUUID, api.apiKey, api.format)
```
Comment (realchoubey) [inline go/dapplooker/dapplookerCharts.go:27]: Get the URL from constant

File: go/dapplooker/dapplookerCharts.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	response, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	defer response.Body.Close()
+
+	if response.StatusCode != http.StatusOK {
```
Comment (realchoubey) [inline go/dapplooker/dapplookerCharts.go:35]: `http.StatusOk`, is 200. Please also check for 201, 202, 203 and 204.

**Review by realchoubey — COMMENTED** [review]
Add chart on which this is tested and its response.

**Comment by realchoubey** [comment]
Add read me. Add example with `output_format` as `json`, and also add sample response with and without output format.

**Comment by 0xSumitBanik** [comment]
> Add chart on which this is tested and its response.

Added in the method definition.

**Comment by 0xSumitBanik** [comment]
> Add read me. Add example with `output_format` as `json`, and also add sample response with and without output format.

I didn't add any README because I am not aware about publishing it in Go Package Dev Repo.
Might need your help on this.

File: go/dapplooker/dapplookerCharts.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		apiKey:    apiKey,
+		format:    format,
+	}
+}
+
+// GetChartData retrieves chart data from the DappLooker API.
+// Returns a slice of maps
+// chartUUID: 0be3ba16-3510-400c-9e9a-95b1a2bf8d97
+// apiKey: xxxxxxxxxxxxxxxxxxxxxxx
+/* Response:
```
Comment (realchoubey) [inline go/dapplooker/dapplookerCharts.go:29]: Add this in readME of go package.

Comment (realchoubey) [reply go/dapplooker/dapplookerCharts.go:29]: @0xSumitBanik I think you missed this.

**Review by realchoubey — APPROVED** [review]
ReadMe will be added later.

**Pull request merged**